tests/test_audio_signal.py

```diff
@@ -391,6 +391,13 @@ def test_magic_getitem():
     npt.assert_allclose(signal[0]._data, time[0])
 
 
+def test_magic_getitem_complex():
+    """Test slicing operations by the magic function __getitem__."""
+    time = np.arange(2 * 3 * 4, dtype=complex).reshape((2, 3, 4))
+    signal = Signal(time, 44100, domain='time', is_complex=True)
+    npt.assert_allclose(signal[0]._data, time[0])
```

Comment:
we could also add a test for TimeData

Reply:
+1 and I would assert `signal[0].complex` because

```
import numpy as np
import numpy.testing as npt

npt.assert_allclose(np.array([1, 1, 1], dtype=complex), [1, 1, 1])
```

does not assert if the Signal is complex